sample_factory/algo/sampling/non_batched_sampling.py

```diff
@@ -125,11 +127,36 @@ def curr_actions(self) -> np.ndarray:
         :return: the latest set of actions for this actor, calculated by the policy worker for the last observation
         """
         actions = ensure_numpy_array(self.last_actions)
-        if self.env_info.integer_actions:
+
+        if isinstance(self.env_info.action_space, gym.spaces.Discrete):
```

Comment:
Looks like this can be just `isinstance(action_space, [Discrete, Box])` instead of two separate ifs

Reply:
or since we're checking for the type here, maybe we can then avoid a type check inside calculate_actions()